.github/do-linux

```diff
@@ -29,6 +29,7 @@ HERE=`dirname "$0"`
 "$HERE"/do-test clang-rv32imafdc "$@"
 "$HERE"/do-zephyr-build arc "$@"
 "$HERE"/do-zephyr-build arc64 "$@"
+"$HERE"/do-zephyr-build microblaze "$@"
```

Comment:
These builds rely on Zephyr SDK 0.15.1, so I don't think we can include this bit yet.

Reply:
Gone.